compliance/aws/disallowed_regions/aws_disallowed_regions.pt

```diff
@@ -37,6 +36,13 @@ parameter "param_disallowed_region" do
   description "List of regions to disallow."
 end
 
+parameter "param_automatic_action" do
+  type "list"
+  label "Automatic Actions"
+  description "When this value is set, this policy will automatically take the selected action(s)"
+  allowed_values ["Terminate disallowed region instances"]
```

Comment:
Let keep the action names short.  use Terminate Instances?

Reply:
updated the policy